$NetBSD: patch-mysys__ssl_CMakeLists.txt,v 1.4 2019/10/14 20:19:29 adam Exp $

Fix build with newer DTrace, this module provides no probes.

--- mysys_ssl/CMakeLists.txt.orig	2019-09-27 07:00:15.000000000 +0000
+++ mysys_ssl/CMakeLists.txt
@@ -53,7 +53,6 @@ TARGET_LINK_LIBRARIES(mysys_ssl ${CMAKE_
 IF(WIN32)
   TARGET_LINK_LIBRARIES(mysys_ssl crypt32)
 ENDIF()
-DTRACE_INSTRUMENT(mysys_ssl)
 
 # MD5_Init() MD5_Update() MD5_Final() are all deprecated
 IF(SOLARIS AND CMAKE_COMPILER_IS_GNUCXX)
